runtime/binding-filesystem/src/main/java/io/aklivity/zilla/runtime/binding/filesystem/internal/FileSystemBindingContext.java

```diff
@@ -64,11 +79,21 @@ public void detach(
         {
             factory.detach(binding.id);
         }
+        if (--bindings == 0)
```

Comment:
Please put a blank line between these `if` statements to avoid a misread as `else-if`.